config/functions

```diff
@@ -72,7 +72,9 @@ print_color() {
   #
   clr_actual="${!clr_name}"
 
-  if [ -z "${clr_actual}" ]; then
+  if [ "${clr_actual}" ]; then
```

Comment:
```suggestion
  if [ -n "${clr_actual}" ]; then
```

Reply:
That is functional equivalent: https://linux.die.net/man/1/test

Reply:
I realise that, but we tend to use `-z` or -`n` as a "house style".

Reply:
OK